### PR TITLE
Enable lint prefer asserts in initializer lists

### PR DESCRIPTION
**Pull request by a14n**
`--assert-initializer` is now required. (Until we switch to Dart 2)

### CODE REVIEW

File: packages/flutter_tools/lib/src/commands/drive.dart [hunk 1]
```diff
@@ -297,7 +297,7 @@ Future<Null> _runTests(List<String> testArgs, String observatoryUri) async {
     ..add('-rexpanded');
   final String dartVmPath = fs.path.join(dartSdkPath, 'bin', 'dart');
   final int result = await runCommandAndStreamOutput(
-    <String>[dartVmPath]..addAll(args),
+    <String>[dartVmPath]..addAll(dartVmFlags)..addAll(args),
```
Comment (Hixie) [inline packages/flutter_tools/lib/src/commands/drive.dart]: fyi @yjbanov

**Comment by Hixie** [comment]
LGTM

File: packages/flutter_tools/lib/src/commands/drive.dart (hunk 1 above)
Comment (yjbanov) [reply packages/flutter_tools/lib/src/commands/drive.dart]: LGTM

**Pull request merged**